src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -17,6 +17,7 @@ trait Warnings {
 
   // Non-lint warnings.
 
+  val warnMacros           = BooleanSetting("-Ywarn-macros", "Enable warnings on macro expansions.")
```

Comment:
Would it be possible to include both unexpanded and expanded code in lint checks?